tests/unit/components/VxpLabel.spec.js

```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { mount } from '@vue/test-utils';
+import VxpLabel from '../../../src/components/VxpLabel.vue';
+import Label from '../../../src/core/components/Label/Label';
+
+describe('VxpLabel.vue', () => {
+  const wrapper = mount(VxpLabel);
+  it('VxpLabel includes a label', () => {
+    expect(wrapper.find(Label).exists()).to.equal(true);
+  });
+});
```

Comment:
here needs to have a test for given text is properly passed to inner Label component.